source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -32,315 +32,136 @@ The procedure on this page documents the following:
-
-      .. code-block:: yaml
-         :class: copyable
-      
-         global:
-            scrape_interval: 60s
-         
-         scrape_configs:
-            - job_name: minio-job-bucket
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/bucket
-              scheme: https
-              static_configs:
-              - targets: [minio.example.net]
-      
-   .. tab-item:: Resources
-
-      .. versionadded:: RELEASE.2023-10-07T15-07-38Z
-
-      The following command scrapes metrics for resources on the MinIO Server.
-
-      .. code-block:: shell
-         :class: copyable
-
-         mc admin prometheus generate ALIAS resource
-
-      Replace :mc-cmd:`ALIAS <mc admin prometheus generate ALIAS>` with the :mc:`alias <mc alias>` of the MinIO deployment.
-
-      .. code-block:: yaml
-         :class: copyable
-
-         global:
-            scrape_interval: 60s
-
-         scrape_configs:
-            - job_name: minio-job-resource
-              bearer_token: TOKEN
-              metrics_path: /minio/v2/metrics/resource
-              scheme: https
-              static_configs:
-              - targets: [minio.example.net]
-      
-- Set an appropriate ``scrape_interval`` value to ensure each scraping operation completes before the next one begins.
-  The recommended value is 60 seconds.
+.. admonition:: Metrics Version 2 Deprecated
+   :class: note
 
-  Some deployments require a longer scrape interval due to the number of metrics being scraped.
-  To reduce the load on your MinIO and Prometheus servers, choose the longest interval that meets your monitoring requirements.
+   Starting with MinIO Server :minio-release:`RELEASE.2024-07-15T19-02-30Z` and MinIO Client :mc-release:`RELEASE.2024-07-11T18-01-28Z`, metrics version 3 replaces the deprecated :ref:`metrics version 2 <minio-metrics-v2>`.
 
-- Set the ``job_name`` to a value associated to the MinIO deployment.
+   MinIO recommends new monitoring configurations use :ref:`version 3 metrics <minio-metrics-and-alerts>`.
 
-  Use a unique value to ensure isolation of the deployment metrics from any others collected by that Prometheus service.
+Collect and alert on metrics
+----------------------------
 
-- MinIO deployments started with :envvar:`MINIO_PROMETHEUS_AUTH_TYPE` set to ``"public"`` can omit the ``bearer_token`` field.
+Generate the scrape configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Set the ``scheme`` to http for MinIO deployments not using TLS.
+Use :mc-cmd:`mc admin prometheus generate --api-version v3 <mc admin prometheus generate --api-version>` to generate a scrape configuration for each :ref:`type of metric <minio-metrics-and-alerts-available-metrics>` you want to scrape with Prometheus.
 
-- Set the ``targets`` array with a hostname that resolves to the MinIO deployment.
+For example, the following command scrapes all version 3 audit metrics for the MinIO cluster:
```

Comment:
All of this "for v3" stuff is clumsy, but I'm concerned users will forget which they are using and follow incorrect instructions. Ideas?